### PR TITLE
Do not set or read user cookies on login.

### PR DESCRIPTION
**Pull request by herdt-michael**
Signed-off-by: Michael Herdt <Michael.Herdt2@bosch-si.com>

### CODE REVIEW

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/login/AbstractHawkbitLoginUI.java [hunk 1]
```diff
@@ -63,6 +33,31 @@
...
+import org.springframework.security.authentication.CredentialsExpiredException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.util.UriComponentsBuilder;
+import org.vaadin.spring.security.VaadinSecurity;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.regex.Pattern;
```
Comment (vigier) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/login/AbstractHawkbitLoginUI.java]: The import order in the code-formatter is misconfigured, should be: `java;javax;org;com;`

Comment (herdt-michael) [reply hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/login/AbstractHawkbitLoginUI.java]: Done

**Comment by vigier** [comment]
👍

**Comment by schabdo** [comment]
@hawkbit-bot verify please

**Pull request merged**